test/ufuzz.js

```diff
@@ -11,6 +11,79 @@
 var vm = require("vm");
 var minify = require("..").minify;
 
+var MAX_GENERATED_FUNCTIONS_PER_RUN = 1;
+var MAX_GENERATION_RECURSION_DEPTH = 15;
+var INTERVAL_COUNT = 100;
+
+var VALUES = [
+  'true',
+  'false',
+  '22',
+  '0',
+  '-0', // 0/-0 !== 0
+  '23..toString()',
+  '24 .toString()',
+  '25. ',
+  '0x26.toString()',
+  '(-1)',
+  'NaN',
+  'undefined',
+  'Infinity',
+  'null',
+  '[]',
+  '({})', // unwrapped the object causes too many syntax errors in statements
+  '"foo"',
+  '"bar"' ];
+
+var BINARY_OPS_NO_COMMA = [
+  ' + ',
+  ' - ',
+  '/',
+  '*',
+  '&&',
+  '||',
+  '^' ];
+
+var BINARY_OPS = [','].concat(BINARY_OPS_NO_COMMA);
+
+var ASSIGNMENTS = [
+  '=',
+  '=',
+  '=',
+  '-=',
+  '/=',
+  '*=',
+  '&=',
+  '|=',
+  '^=',
+  '+=' ];
+
+var UNARY_OPS = [
+  '--',
+  '++',
+  '~',
+  '!',
+  'void ',
+  'delete ', // should be safe, even `delete foo` and `delete f()` shouldn't crash
+  ' - ',
+  ' + ' ];
```

Comment:
We can probably use a generator for that. I'm not sure if UglifyJS optimizes certain comparisons away? I mean, `n=0; typeof n === 'number'` kind of silliness could be eliminated. If there's no support for those kinds of artifacts then a generator is probably not needed either.